flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/RestartPipelinedRegionFailoverStrategy.java

```diff
@@ -196,6 +196,7 @@ public RestartPipelinedRegionFailoverStrategy(
 	 * @return the failover region that contains the given execution vertex
 	 */
 	@VisibleForTesting
+	@Deprecated
```

Comment:
Seems this deprecation commit is not needed since the method will be just removed in a following commit?

Reply:
I will remove this commit before merging. It was for myself when I started working on this ticket.